docs/user-guide/project.md

```diff
@@ -28,120 +28,218 @@ A user menu with different actions is accessible for each member of the project
+## Data model
 
-Project admin can create one or several permission groups in their project to set specific permissions on data to a user group.
+The definition of the data model (*ontology*) is the most important step. The data model is indispensable for structuring your data. Our platform provides a tool for an easy creation of one or more project data models. First, you have to know which data and sources you want to work with. The data model can be flexible and customizable. 
 
----
+The question which you have to answer before you create your data model is **according to which criteria do you organize your data**? In this respect it may be useful to ask yourself:
 
-## Data model
+- *How are your data organized?*
+- *What are the goals you want to achieve, which research questions do you want to answer?*
 
-The most important step in the project is the definition of the data model. DSP-APP will provide a tool for easy creation of data models. First, you have to know which data and sources you want to work with. The data model can be flexible and customizable. With DSP-APP, you can comply with the FAIR data standard, but compliance is not required to analyze your data.
+As soon as you have come to a conclusion concerning the structure of your data, you're all set to create your data model.
 
-The questions to answer in creating your data model:
 
-- *What kind of data do I have in my project?*
-- *What are the sources and what are their metadata?*
+### Create your data model
 
-**For example:** You have interviewed 20 people. During these interviews that you recorded, you talked about photographs. Among all the data collected during the project, the most important are:
+**Go to *Data model* and click *New data model***
 
-- audio-files of the interview
-- transcribed text of conversations (or transcribe the files within the web application)
-- photographs
-- data about the person you interviewed
-- location where the photograph was taken
+Go to your project, select the tab `Data model` (step 1) and click the button `New data model` (step 2) as shown below:
+![Data model.](../assets/images/Fig2.png)
 
-Diagram 1 shows the relationships of the data by resource classes from these experiences.
+By clicking `New data model`, a dialogue box opens:
+![Define new data model.](../assets/images/Fig3.png)
 
-![Relationship of the data by resource classes](../assets/images/diagram-data-model.png)*Relationship of the data by resource classes*
 
-### Create your data model
+**Create data model**
 
-Go to your project, select the tab `Data model` and click on button `New data model` or `Create your first data model` (in case of a brand new project). This will open a form in a dialog box where you have to set a unique name and a label. Optional you can also add a short description of form of a comment.
+Now you've to set a unique name (*please consider the NOTE*) and you can add a comment. Push the button `Create` to create your data model.
```

Comment:
I would split "you've" into "you have". This could just be my American English though.